src/test/analyzer_test.ts

```diff
@@ -85,6 +85,38 @@ suite('Analyzer', () => {
           assert.deepEqual(elements.map((e) => e.tagName), ['my-element']);
         });
 
+    test('analyzes inline scripts correctly', async() => {
+      const document = await analyzer.analyze(
+          'static/inline-documents/inline-documents.html');
+      const jsDocs = document.getByKind('js-document');
+      assert.equal(jsDocs.size, 1);
+      const jsDoc = jsDocs.values().next().value;
+      assert.deepEqual(jsDoc.sourceRange, {
+        file: 'static/inline-documents/inline-documents.html',
+        start: {line: 3, column: 2},
+        end: {line: 5, column: 11}
+      });
+      assert.isObject(jsDoc.astNode);
+      assert.equal(jsDoc.astNode!.tagName, 'script');
+      assert.equal(jsDoc.astNode!.parentNode!.tagName, 'head');
```

Comment:
Chose it as a simple proxy for checking that the astNode was accurate. But having worked with the analyzer more now I feel we can assume that if the astNode is set it is accurate (at least in a larger analyzer test like this). So I'll back this out.